jobs/prepare_data.py

```diff
@@ -76,11 +80,15 @@ def main(starttime, hstart, hstop, cfg):
     """
 
     if cfg.target is tools.Target.ICON or cfg.target is tools.Target.ICONART or \
-       cfg.target is tools.Target.ICONARTOEM:
+       cfg.target is tools.Target.ICONARTOEM or cfg.target is tools.Target.ICONARTGLOBAL:
 
         logging.info('ICON input data (IC/BC)')
 
         starttime_real = starttime + timedelta(hours=hstart)
+        time = starttime + timedelta(hours=hstart)
+        year = time.year
+        month = time.month
+        day = time.day
```

Comment:
You can remove `time` here and just use `starttime_real` to get the year/month/day info. The variable `time` isn't used further on (as far as I can see).

Reply:
Done in [8d3afb2](https://github.com/C2SM/processing-chain/pull/16/commits/8d3afb2c95e4bf7b2a01efe54253edb43f215316)